site/load-chart.js

```diff
@@ -5,6 +5,7 @@ google.charts.setOnLoadCallback(setupChart)
 
 const fetchData = fetch('scores.json')
 const embed = location.search === '?embed'
```

Comment:
I guess we can remove this and also the logic in lines 17-19 for adding the `embed` class.

Reply:
We need them to hide things from the page.

Reply:
Oh, you are right.